styles/stylesheet.css

```diff
@@ -0,0 +1,374 @@
+.header {
+  height: 300px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  background-image: image("images/header-image.jpg");
+  overflow: hidden;
+}
+
+.header-img {
+  max-width: 100%;
+}
+
+.header-logo {
+  text-align: left;
+  position: absolute;
+  top: 20px;
+  left: 60px;
+}
+
+.header-checkout {
+  text-align: left;
+  position: absolute;
+  top: 20px;
+  right: 100px;
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  text-decoration: none;
+  color: #fff;
+}
+
+.header-checkout img {
+  padding-right: 10px;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: 4px 8px 4px 8px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 240px;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+nav {
+  background-color: #ccc;
+  margin-top: -16px;
+}
+
+ul {
+  list-style-type: none;
+  padding: 8px 0px 8px 0px;
+  margin-left: 60px;
+}
```

Comment:
I would also recommend using rem units instead of px for padding etc for a more scalable layout.